rack-protection/rack-protection.gemspec

```diff
@@ -39,9 +39,6 @@ RubyGems 2.0 or newer is required to protect against public gem pushes. You can
   s.required_ruby_version = '>= 2.6.0'
 
   # dependencies
-  s.add_dependency 'base64'
+  s.add_dependency 'base64', '>= 0.1.0'
```

Comment:
It is still open-ended 😄  (same with `multi_json` in sinatra-contrib)

Not sure we need to act, hopefully conservative gems that don't change the API anytime soon (if ever?)